### ISSUE TITLE
[Bug] Minification causes bug in ace.js: Spaces replaced with tabString

### ISSUE THREAD

**Issue by GlitchHound** [ajaxorg/ace#2958]
Symptom:

Pressing space would replace the inserted " " with the value of session.getTabString() even with useSoftTabs set to false. This means one press of the spacebar would result in multiple spaces.
Bug was not present with unminified source (both minified and unminifed files pulled from ajaxorg/ace-builds).

Cause:
On line 974 of ace/lib/ace/editor.js, in the function definition for "insert", the tab literal "\t" is replaced during minification with " ". This causes the conditional to evaluate to true.

Issue added mostly in case someone else runs into this; debugging was frustrating.

#2957 is a simple proposed fix.

**Comment by GlitchHound** [comment]
In the shower it occurred to me that this probably wouldn't have gone unnoticed and there must be some other explanation.

Turns out there was. Somewhere along the line in my infinite wisdom I decided to copy one of these files, paste it, and save that. Downloading them doesn't have the missing tab issue.

TL;DR: Don't copy and paste code, kids.
